MVC/models.js

```diff
@@ -30,3 +30,27 @@ exports. fetchArticle = (article_id) => {
     });
 }
 
+exports. fetchArticles = () => {
+    return db.query(
+        // I had lots of difficulty here: solution was it HAD to be groupped by articles.article_id
+        // :: reformats the string into the specified INT, As relabels that column (so property)
+        `SELECT
+            articles.author,
+            articles.title, 
+            articles.article_id,
+            articles.topic,
+            articles.created_at,
+            articles.votes,
+            articles.article_img_url,
+        COUNT(comment_id) :: INT AS comment_count
+        FROM articles
+        LEFT JOIN comments
+        ON comments.article_id = articles.article_id
+        GROUP BY articles.article_id
+        ORDER BY articles.created_at DESC
+    ;`)
```

Comment:
Great job on this query, it's a tricky one! Same as before lets remove comments and logs before committing. Feel free to leave messages for us in the commit if you want to or drop us a message if there's anything to communicate :)